<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5314" started-at="2014-04-12T06:58:55Z" finished-at="2014-04-12T06:59:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5314" started-at="2014-04-12T06:58:55Z" finished-at="2014-04-12T06:59:00Z">
      <class name="com.example.tests.TestFolderCreation">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.TestFolderCreation@1aa7a6a]" name="setUp" is-config="true" duration-ms="47" started-at="2014-04-12T09:58:55Z" finished-at="2014-04-12T09:58:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testFolderCreation()[pri:0, instance:com.example.tests.TestFolderCreation@1aa7a6a]" name="testFolderCreation" duration-ms="5215" started-at="2014-04-12T09:58:55Z" finished-at="2014-04-12T09:59:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
Expected: <Folders [[Persons, newfolder]]]]><![CDATA[>
     but: was <Folders [[Persons]]><![CDATA[]]]]><![CDATA[>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
Expected: <Folders [[Persons, newfolder]]]]><![CDATA[>
     but: was <Folders [[Persons]]]]><![CDATA[>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.example.tests.TestFolderCreation.testFolderCreation(TestFolderCreation.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFolderCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.TestFolderCreation@1aa7a6a]" name="tearDown" is-config="true" duration-ms="0" started-at="2014-04-12T09:59:00Z" finished-at="2014-04-12T09:59:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.TestFolderCreation -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
